target/linux/ath79/dts/ar9342_ubnt_nanobeam-ac-gen2.dts

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include "ar9342_ubnt_wa.dtsi"
+
+/ {
+	compatible = "ubnt,nanobeam-ac-gen2", "ubnt,wa", "qca,ar9342";
+	model = "Ubiquiti NanoBeam AC Gen2 (WA)";
```

Comment:
Unless there are devices with the same name and another board, I'd drop the (WA) here.

Reply:
There is the XC model with the same name. https://openwrt.org/toh/ubiquiti/ubiquiti_nanobeam_ac

Reply:
So, we essentially need
ar9342_ubnt_nanobeam-ac-gen2-wa.dts
and
qca9558_ubnt_nanobeam-ac-gen2-xc.dts
then?!

Reply:
Hmmm, yeah?
Same would hold for ar9342_ubnt_nanobeam-ac.dts that has a wa and xc variant.

Reply:
Well, then same should have been done for the gen1.

Why do they have two variants for each generation?

Reply:
No idea. ^^ I think no XC platform is supported from OpenWrt... (?)

Reply:
So, you would prefer not adding the suffix because you were first, and then having the one supporting the second device have to care about it?

Reply:
I prefer something that looks consistent... :S 
Look at `ubnt_nanostation-loco-m-xw` and `ubnt_nanostation-loco-m`.

Do u prefer if I rename to  ar9342_ubnt_nanobeam-ac-gen2-wa.dts? I don't mind changing it the way u want. :)

Reply:
> Look at ubnt_nanostation-loco-m-xw and ubnt_nanostation-loco-m.

Exactly, that's one nice example where it went wrong.

> Do u prefer if I rename to ar9342_ubnt_nanobeam-ac-gen2-wa.dts? 

Yes, or why do you think I asked you to do it?

Reply:
Sorry, I didn't get that. Thought it was more of a question. Changed it. ;)